modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -189,19 +194,19 @@ Because the following endpoints use mutual TLS authentication to establish the c
 ** `configuration-resolver.prod-eu.msap.io`
 
 [%header%autowidth.spread]
-.Port IP Addresses and Hostnames to Whitelist
+.Port IP Addresses and Hostnames to add to the allow list
 |===
 | Port | Protocol | Hostnames | Description
 .2+| 443 .2+| AMQP over WebSockets | US control plane: `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the control plane.
 | EU control plane: `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
+.2+| 443 (v1.8.50, or later) .2+| TCP (Lumberjack) | US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
+| EU control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
```

Comment:
The EU control plane hostname is the same as the US control plane.. I think that might be an error.
cc @sarjeet2013 to confirm.

Reply:
yes, it's an error. EU endpoint is: `dias-ingestor-router.eu-central-1.prod-eu.msap.io`

Reply:
Updated.